docs/source/conf.py

```diff
@@ -23,7 +23,8 @@
 #sys.path.insert(0, os.path.abspath('.'))
 
 if os.environ.get('READTHEDOCS', ''):
-    # RTD doesn't use the Makefile, so re-run autogen_config.py here.
+    # RTD doesn't use the repo's Makefile to build docs. Run autogen_config.py
+    # here to generate rst files from ipynb files.
```

Comment:
Ideally we should also run the ipynb -> rst conversion here - see #156 - but I think that needs pandoc installed on RTD. Maybe now that RTD supports conda, we can [get that](http://anaconda.org/asmeurer/pandoc) - but that's definitely a matter for another PR